roles/cpu_vulnerabilities/tasks/main.yml

```diff
@@ -7,5 +7,5 @@
 
 - name: set the collected cpu vulnerabilities info as facts
   set_fact:
-    stockpile_cpu_vulnerabilities: "{{ vulnerabilities_output.stdout_lines }}"
+    stockpile_cpu_vulnerabilities: "{{ vulnerabilities_output.stdout }}"
   when: vulnerabilities_output.stdout_lines is defined
```

Comment:
`stdout_lines` was removed, but still used here?

Reply:
I send the data w/o splitting lines to make scribe to print only one document containing all CPU vulnerabilities

splitting lines in stockpile -> https://termbin.com/g8ac
PR's implementation, w/o splitting lines in stockpile -> https://termbin.com/cew4